torch/csrc/jit/runtime/static/passes.cpp

```diff
@@ -950,6 +931,149 @@ void FuseListUnpack(std::shared_ptr<torch::jit::Graph>& graph) {
   }
 } // namespace jit
 
+
+// Similar to FuseListUnpack with two differences
+// Its only for fb::equally_split operator
+// the operators results are being accessed by get_item rather than ListUnpack
+void FuseEquallySplitGetItemUnpack(std::shared_ptr<torch::jit::Graph>& graph) {
+  // replacement contains (old_node, new_node, list_unpack_node)
+  struct UserToRemove{
+    Node* user;
+    ArrayRef<Value*> user_outputs;
+  };
+
+  struct Replacement
+  {
+    Node* node_to_be_fused;
+    c10::Symbol new_sym;
+    std::vector<UserToRemove> user_to_remove;
+  };
+
+  std::vector<Replacement> replacement_list;
+
+  DepthFirstGraphNodeIterator graph_it(graph);
+  const auto equally_split = fromQualString("fb::equally_split");
+  const auto get_item = fromQualString("aten::__getitem__");
+  for (auto node = graph_it.next(); node != nullptr; node = graph_it.next()) {
+    if (node->kind() != equally_split) {
+      continue;
+    }
+
+    const Value* value_out = node->outputs()[0];
+
+    const auto new_sym = fromQualString("static_runtime::fused_equally_split");
+
+    const Value* value_num_splits = node->inputs()[1];
+
+    // check if equally_split num of splits is equal to the number of output uses
+    if(!toIValue(value_num_splits).has_value() || !toIValue(value_num_splits).value().isInt() || toIValue(value_num_splits).value().toInt() != value_out->uses().size()) {
+      continue;
+    }
+
+    bool should_fuse = true;
+
+    std::vector<UserToRemove> user_to_remove_list;
+
+    std::unordered_set<std::int64_t> getItem_indices;
+
+    for(auto index = 0; index < value_out->uses().size(); index++) {
```

Comment:
nit: please do not put so many blank lines; keep style consistent with the surrounding code

Reply:
removed the not needed blank lines